MetaMask/metamask-extension
# Update Smart Swaps

## **Description**

Since we are making backend changes for Smart Swaps, we need to ask users again for Smart Swap opt-in. This PR also includes some UI / content changes for Smart Swaps.

## **Pending Items**
- [x]  Fix the newly updated STX controller, so transactions will show up in the Activity list
- [x] Update content for the Smart Swap tooltip

## **Manual testing steps**

1. In the extension open this file: node_modules/@metamask/smart-transactions-controller/dist/constants.js
2. Make sure that the API_BASE_URL is pointing to a dev server
3. Open the extension
4. Click on Swap
5. You will see our updated Opt In modal
6. You can submit a smart swap after you opt in

## **Screenshots/Recordings**
![image](https://github.com/MetaMask/metamask-extension/assets/80175477/c461f38c-fa82-4e2d-88e0-3304af1085d5)
